src/window.rs

```diff
@@ -838,6 +838,63 @@ impl Window {
     self.window.set_maximizable(maximizable)
   }
 
+  /// Set the taskbar progress
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Linux:** May work, might not work
+  /// - **macOS/Android/iOS:** Unimplemented
+  #[cfg(windows)]
+  pub fn set_taskbar_progress(&self, current: u64, total: u64) {
+    self.window.set_taskbar_progress(current, total);
+  }
+
+  #[cfg(target_os = "linux")]
+  pub fn set_taskbar_progress(&self, current: u64, total: u64, unity_uri: Option<&str>) {
+    let uri = match unity_uri {
+      Some(a) => Some(a.to_string()),
+      _ => None,
+    };
+    self.window.set_taskbar_progress(current, total, uri);
+  }
+
+  /// Set the taskbar progress
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Linux:** May work, might not work
+  /// - **macOS/Android/iOS:** Unimplemented
+  #[cfg(windows)]
+  pub fn set_taskbar_progress_state(&self, state: TaskbarProgressState) {
+    let taskbar_state = {
+      match state {
+        TaskbarProgressState::None => 0,
+        TaskbarProgressState::Intermediate => 1,
+        TaskbarProgressState::Normal => 2,
+        TaskbarProgressState::Paused => 4,
+      }
+    };
+
+    self.window.set_taskbar_progress_state(taskbar_state);
+  }
+
+  #[cfg(target_os = "linux")]
+  pub fn set_taskbar_progress_state(&self, state: TaskbarProgressState, unity_uri: Option<&str>) {
```

Comment:
ditto

Reply:
?

Reply:
you should not have methods with different signature based on the platform here

Reply:
You mean to add it in **WindowExtWindows** and the other one for the linux one?

Reply:
yup